src/tests/unit_asio_stream.cpp

```diff
@@ -736,5 +736,5 @@ BOTAN_REGISTER_TEST("tls", "tls_asio_stream", Asio_Stream_Tests);
 
 }  // namespace Botan_Tests
 
-   #endif  // BOOST_VERSION
-#endif     // BOTAN_HAS_TLS && BOTAN_HAS_BOOST_ASIO
+   #endif
+#endif  // BOTAN_HAS_TLS && BOTAN_HAS_BOOST_ASIO
```

Comment:
```suggestion
#endif  // BOTAN_HAS_TLS && BOTAN_HAS_TLS_ASIO_STREAM
```

Reply:
Mhh, somehow, I thought `clang-format` was responsible for that. Fixed.